projects/plugins/jetpack/modules/shortcodes/descript.php

```diff
@@ -87,6 +87,20 @@ function jetpack_shortcodereverse_descript( $content ) {
 	return $content;
 }
 
+/**
+ * Checks if dnt (Do not track) is enabled.
+ *
+ * @return bool
+ */
+function jetpack_descript_is_dnt_enabled() {
```

Comment:
We have something similar already:

https://github.com/Automattic/jetpack/blob/e50c798ac77031439d8cb6a4a632d29f70dfbe53/projects/plugins/jetpack/modules/stats.php#L95-L100

What do you think about extracting this to a shared location, maybe `projects/plugins/jetpack/functions.global.php` for now?